tests/rasterizers/test_rasterizer_caching.py

```diff
@@ -57,11 +57,8 @@ def test_data_caching(input_files, ext, shape, target, features):
             rasterizer, cache_kwargs={'cache_pattern': cache_pattern}
         )
 
-        assert rasterizer.shape[:3] == (
-            shape[0],
-            shape[1],
-            rasterizer.shape[2],
-        )
+        good_shape = (shape[0], shape[1], rasterizer.shape[2])
```

Comment:
lol. fair point :)

Reply:
haha i dont even remember why i felt the need to fix this but yes trying to have less ugly one liners given the new linting.